src/styles/reviews.scss

```diff
@@ -0,0 +1,16 @@
+.rewiew {
```

Comment:
The class name 'rewiew' appears to be a typo. It should likely be 'review' to match the corrected class name in the HTML.